boards/native/native_sim/Kconfig

```
@@ -50,6 +51,13 @@ config NATIVE_POSIX_SLOWDOWN_TO_REAL_TIME
 	  Transitional option which allows applications which targeted native_posix
 	  to set the correct native_sim option (CONFIG_NATIVE_SIM_SLOWDOWN_TO_REAL_TIME)
 
+config NATIVE_SIM_REBOOT
+	bool "Reboot native sim"
+	depends on REBOOT
+	default y
```

```suggestion
	default n
```
This does not work with valgrind, and I guess with some other tools.
And I'd expect that a few developers only trigger reboots in error conditions (so they will prefer native_sim to error out), so I'm inclined to leave the default as n.

---

I changed it. I did not have this use case in mind, I'm using native_sim to test behaviour where the real hardware also does a reboot.